tasks/main.yml

```diff
@@ -15,9 +15,18 @@
 - name: Add Docker repository.
   apt_repository: repo='deb https://get.docker.io/ubuntu docker main' state=present
 
-- name: Install packages
-  apt: name="{{ item }}" update_cache=yes
-  with_items: packages
+- name: Install support packages
+  apt: name="{{ item }}"
+  with_items:
+    - wget
+
+- name: Install docker package
+  apt: name="{{ item }}{% if docker_version is defined %}={{ docker_version }}{% endif %}"
+  with_items:
+    - lxc-docker
+
+- name: Do not use sysv to start docker daemon
```

Comment:
I can't reproduce this problem. Neither on my Tsuru environment or the Vagrant machine in this PR with fb194af1ef44d8611ed65fd5455508635fd258ba reverted. But you're not the first person to come across this, so I don't understand..